EventPlaning.BL.Tests/EventManagerTests.cs

```diff
@@ -162,91 +188,5 @@ public void GetEventIdFail()
         {
             Assert.IsNull(eventManager.GetEventId("Mono"));
         }
-
-        //[TestMethod]
-        //public void CreateActivityOK()
-        //{
-        //    var activity = new ActivityData()
-        //    {
-        //        Name = "Pandemic",
-        //        ActivityType = "Board game",
-        //        Description = "Save the world!",
-        //        MinParticipants = 1,
-        //        MaxParticipants = 4,
-        //    };
-
-        //    Assert.IsTrue(eventManager.CreateActivity(activity));
-        //}
-
-        //[TestMethod]
-        //public void CreateActivityFail()
-        //{
-        //    var activity = new ActivityData()
-        //    {
-        //        Name = "Battle city",
-        //        ActivityType = "Board game",
-        //        Description = "Save the world!",
-        //        MinParticipants = 1,
-        //        MaxParticipants = 4,
-        //    };
-
-        //    Assert.IsFalse(eventManager.CreateActivity(activity));
-        //}
-
-        //[TestMethod]
-        //public void CreateActivityTypeOK()
-        //{
-        //    var activityType = new ActivityTypeData()
-        //    {
-        //        Name = "Puzzle",
-        //    };
-
-        //    Assert.IsTrue(eventManager.CreateActivityType(activityType));
-        //}
-
-        //[TestMethod]
-        //public void CreateActivityTypeFail()
-        //{
-        //    var activityType = new ActivityTypeData()
-        //    {
-        //        Name = "Board game",
-        //    };
-
-        //    Assert.IsFalse(eventManager.CreateActivityType(activityType));
-        //}
-
-        //[TestMethod]
-        //public void CreateEventOK()
-        //{
-        //    var eventData = new EventData()
-        //    {
-        //        Name = "Board game challenge",
-        //        Date = "2018-02-08",
-        //        Address = "Ateities g. 4F, Vilnius",
-        //        Duration = "5",
-        //        MinParticipants = 4,
-        //        MaxParticipants = 12,
-        //        Activities = new string[1] { "Monopoly" },
-        //    };
-
-        //    Assert.IsTrue(eventManager.CreateEvent(eventData));
-        //}
-
-        //[TestMethod]
-        //public void CreateEventFailNameExists()
-        //{
-        //    var eventData = new EventData()
-        //    {
-        //        Name = "BG and VG event",
-        //        Date = "2018-02-08",
-        //        Address = "Ateities g. 4F, Vilnius",
-        //        Duration = "5",
-        //        MinParticipants = 4,
-        //        MaxParticipants = 12,
-        //        Activities = new string[1] { "Monopoly" },
-        //    };
-
-        //    Assert.IsFalse(eventManager.CreateEvent(eventData));
-        //}
```

Comment:
not useful?

Reply:
As we talked, any records creating methods can't be tested by using unit tests. Later will add integration tests.

Reply:
👍